vendor/github.com/google/go-github/v31/github/github-accessors.go

```diff
@@ -11332,6 +11332,16 @@ func (r *RepositoryPermissionLevel) GetUser() *User {
 	return r.User
 }
 
+// GetAssertsURL returns the AssetsURL field if it's non-nil, zero value otherwise. Created to generate a deprecation warning
```

Comment:
this custom method can be safely omitted here as we can reuse `GetAssetsURL()` to populate the misspelled attribute

Reply:
My intent with the extra method was to have a place to handle the deprecation message. I haven't made the time to go through the deprecation docs you linked, but hopefully I'll get to that soon